the-collab-lab/tcl-35-smart-shopping-list
# Aa styling homepage

## Description

Related to PR #37, this PR is to refactor some Bootstrap and CSS code.
Others remain the same.

To fulfill the AC:
- Welcome page look presentable, and agreed by all team members.
- Decided color is default Bootstrap's `success` color.
- This app included a button to create a list and a button to join an existing list.

Additional info:
- App functionality to join existing list & to create a new list are working as they suppose to.
- The page and landmarks are readable by screen readers (NVDA). 
- Home page has 0 accessibility issues based on audit on axe devtools, WAVE & Lighthouse.
![axe-devtools-homepage](https://user-images.githubusercontent.com/45172775/143661704-b60f9fbd-dd4c-44a4-8e19-68bd9e7f0db6.jpg)




## Related Issue

Closes #38 



## Acceptance Criteria

- Welcome page should look presentable
- Should incorporate decided colors
- Should include a button to create a list and join an existing list



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before




### After




## Testing Steps / QA Criteria

- Pull the branch to your machine.
- Run npm install to install all dependencies.
- Run npm start to start the project and see the style on the browser localhost:3000.
- Drag the browser or use the browser's devtools by toggling the device toolbar to see the responsiveness.
- Use WAVE, or axe DevTools, or Lighthouse and screen readers to check on the accessibility.
